app/themes/dashboard/ui/styles/theme.less

```diff
@@ -43,3 +43,11 @@ z-index: 10;
 .pre-xml {
   background-color: #ecf0f5;
 }
+
+.nav-pills>li>a {
+  border-radius: 4px !important;
+}
+
+.nav-pills>li.active>a {
+  background-color: #384246 !important;
```

Comment:
Yes, @grtjn and I tested it today and I'd prefer we didn't change the color of those pill buttons. I think that should be a choice made by the engineer using the template.